src/api/routes.js

```diff
@@ -0,0 +1,28 @@
+var req_handlers = require("./handlers.js").request_handlers()
+
+var appRouter = function (app) {
+	app.get("/api/v1.0/hypothesis/:hypothesis_id/res", function (req, res) {
```

Comment:
res might not be required here

Reply:
res for sending back the status of the query and checking if the endpoint is online

Reply:
that shouldn't be on the end-point route
rather it should be in the response body

Reply:
OK I thought you were referring to the variable res

Reply:
Ok I thought you were referring to the res variable